torchbenchmark/models/resnet50/__init__.py

```diff
@@ -1,29 +1,37 @@
 
-# Generated by gen_torchvision_benchmark.py
 import torch
 import torch.optim as optim
 import torchvision.models as models
 from ...util.model import BenchmarkModel
 from torchbenchmark.tasks import COMPUTER_VISION
 
-#######################################################
-#
-#       DO NOT MODIFY THESE FILES DIRECTLY!!!
-#       USE `gen_torchvision_benchmarks.py`
-#
-#######################################################
+from torchbenchmark.util.env_check import parse_extraargs
+from torchbenchmark.util.fx2trt import lower_to_trt
```

Comment:
Just to be safe, maybe let's put the import inside `if self.extra_args.fx2trt:` check?

Reply:
Right, fixed.